src/app/modules/gb-cohort-selection-module/constraint-components/gb-combination-constraint/gb-combination-constraint.component.ts

```diff
@@ -155,4 +155,8 @@ export class GbCombinationConstraintComponent extends GbConstraintComponent impl
     (<CombinationConstraint>this.constraint).dimension = this.selectedDimension;
   }
 
+  getDimensionIcon(dimension: string): string {
+    return this.constraintService.getDimensionIcon(dimension);
```

Comment:
I don't understand why there are so many service calls in the PR for fecthing an icon class. Why not create one static helper function in a IconHelper class or something like that?

Reply:
done!